portal-impl/src/com/liferay/portal/service/persistence/impl/UserFinderImpl.java

```diff
@@ -756,39 +1047,48 @@ else if (group.isUserGroup()) {
 						"usersOrgsTree",
 						new ArrayList<Organization>(organizations.values()));
 				}
+
+				paramsMapList.add(params2);
```

Comment:
@jorgediaz-lr 

For your reference, the paramsMapList is not populated correctly. We should only have one params2 in the list. On line 1112, if the condition passes, we need to override the params2 value that's being set here. params2 should not be added to the list here. With your changes, we can end up with two params2 in the list.